### PR TITLE
Assignment 2

### PR DESCRIPTION
**Pull request by AvaEN**

### CODE REVIEW

File: assignment2.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Ava N.
+June 29, 2016
+
+1.  There can be two or more operators in a single line of program code as long as there is a logical operator seperating the relational operator statements.
```
Comment (oldclesleycode) [inline assignment2.txt:4]: right, so long as each operator has the required amount of operands!

File: assignment2.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Ava N.
+June 29, 2016
+
+1.  There can be two or more operators in a single line of program code as long as there is a logical operator seperating the relational operator statements. 
+
+2. Syntax errors are incorrect spellings of reserved words or incorrect puncuation. Logical errors give an output that is unexpected or unwanted.
```
Comment (oldclesleycode) [inline assignment2.txt:6]: Correct - it's important to note that logical errors occur during run time, whereas syntax errors occur during compile time.

File: assignment2.txt [hunk 3]
```diff
@@ -0,0 +1,14 @@
+Ava N.
+June 29, 2016
+
+1.  There can be two or more operators in a single line of program code as long as there is a logical operator seperating the relational operator statements. 
+
+2. Syntax errors are incorrect spellings of reserved words or incorrect puncuation. Logical errors give an output that is unexpected or unwanted. 
+
```
Comment (lesley2958) [inline assignment2.txt]: Correct, but remember that boolean values don't exist in C! So the answer is actually 1!

File: assignment2.txt [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+3. 10>=10 && 10<25 && 10 !=12
```
Comment (oldclesleycode) [inline assignment2.txt:8]: Correct, but remember that boolean values don't exist in C! So the answer is actually 1!

File: assignment2.txt [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+     true    true     true
+	  answer=true
+
+4. The given example is incorrect because there cannot be more then one relational operator in a single statement. This is only allowed, however, when there is a logical operator inbetween them. The statement can instead look like this: 0<x && x<15. This represents what x has to be equal to under these circumstances.
```
Comment (oldclesleycode) [inline assignment2.txt:12]: Correct! 💯

File: assignment2.txt [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+
+2. Syntax errors are incorrect spellings of reserved words or incorrect puncuation. Logical errors give an output that is unexpected or unwanted. 
+
+3. 10>=10 && 10<25 && 10 !=12
+     true    true     true
+	  answer=true
+
+4. The given example is incorrect because there cannot be more then one relational operator in a single statement. This is only allowed, however, when there is a logical operator inbetween them. The statement can instead look like this: 0<x && x<15. This represents what x has to be equal to under these circumstances. 
+
+5. Example i is invalid because 'then' should not be written. Example ii is invalid because there needs to be parentheses around a>b. Example iii is valid. Example iv is invalid because there needs to be a semicolon(;) after c=0.
```
Comment (oldclesleycode) [inline assignment2.txt:14]: Correct! 💯

File: calculator.c [hunk 7]
```diff
@@ -0,0 +1,48 @@
+/*Ava N.*/
```
Comment (oldclesleycode) [inline calculator.c:1]: Perfect job! The only thing I'd suggest is to have more comments!

File: randnum.c [hunk 8]
```diff
@@ -0,0 +1,31 @@
+/*Ava N.*/
```
Comment (oldclesleycode) [inline randnum.c:1]: Perfect job! 💯

File: user_input.c [hunk 9]
```diff
@@ -0,0 +1,31 @@
+/*Ava N.*/
```
Comment (oldclesleycode) [inline user_input.c:1]: Perfect job! 💯

**Pull request closed without merging**